### ISSUE TITLE
Issues with exporting.

### ISSUE THREAD

**Issue by hemanth** [hakimel/reveal.js#1363]
Not sure if this is the right place to talk about slides but after exporting these [slides](http://slides.com/gnumanth/learnit/fullscreen?token=zZ84CDnDw7kTcyFz6YQgYSgh2sAr) I am not able to see the code content at all!

**Comment by dvenkatsagar** [comment]
What did you do exactly....??

**Comment by hemanth** [comment]
Blank divs wherever the code is and some HTML!

**Comment by dvenkatsagar** [comment]
The thing is slides.com doesnt allow exporting to pdf.
The process that I did was to create a clone of reveal.js project and then copy the html from the export section in the slides.com editor and then add some config options and dependencies for printing pdf.

**Comment by hemanth** [comment]
Did the same.

**Comment by dvenkatsagar** [comment]
Well one thing for sure is that it is not a problem of reveal.js but i think it would be a problem with slides.com.

**Comment by hemanth** [comment]
Hmm...is there a repo for slides.com ?

**Comment by dvenkatsagar** [comment]
No i think, Its still under progress
you might want to check their forums.

**Comment by seefood** [comment]
And where are those forums? I only found a knowledgebase and a place to give suggestions, not interactive support like here...

I also created the skeleton of the lecture online and wanted to move on to edit it offline. I found the output file is unindented, won't pop up the prompter when I run it with "grunt serve" and even section tags are not easily found so slides are not evenly distributed, nor human editable. if I run it through tidy, it kills all the "section" and other tags and it's unusable. this is really annoying, since editing all this in VIM is already painful, but I use it with Hebrew slides, and that's REALLY not fun in VIM...

This time I'll just have to give up and painfully reshape and reindent 40 slides by hand. I hope @hakimel has improvements to offer in the future.

**Comment by dvenkatsagar** [comment]
Ok, I have one final question, did you guys check the beta features(in the settings of slides.com) because the new deck editor has lots of improvements and that is the one that worked for me.

@hemanth. Try out the beta feature extended version. If you did try it out, and it still doesnt work, then download the presentation as a HTML file(go to export -> download via HTML. (Only in beta extended version)). Go to unminify.com and unminify it. See if you can use that. If all that doesnt work, just send me the link to your presentation, and ill experiment with it.

And @seefood. Yes in those suggestion, there are comments that are useful. And for the intendation thing, try this [http://www.freeformatter.com/html-formatter.html](http://www.freeformatter.com/html-formatter.html). and check if it works. The problem with tidy is that it doesnt support html5, and section tags are new in html5.

And actually this is a problem with slides.com, you might want to contact their support and talk to them.

**Comment by hemanth** [comment]
@dvenkatsagar I am not able to see the beta feature extended version, I have mentioned a link to my slides in the first comment.

**Comment by dvenkatsagar** [comment]
@hemanth I mean in the profile settings like this [http://i.imgur.com/AWwiWkd.png](http://i.imgur.com/AWwiWkd.png)

**Comment by seefood** [comment]
well, this is WITH the new editor.

I ended up exporting the html, getting rid of the embedded  scripts in favor of the regular linked one from the reveal.js github, using two SED commands to put all section and /section tags on seperate lines, and finally breaking up the settings JSON and indenting everything with VIM's built in indentation logic (works better than any CLI I tried). I can now finally move on to edit this some more. All this had to be done because I'm missing a few features from the slides.com site. sad.

bottom line - reveal.js with RTL languages is a bit painful, and exporting from slides.com can use some smoother edges. I'll do the next RTL presentations with other tools.

**Comment by hemanth** [comment]
@dvenkatsagar I was already under New deck editor! Yes it's private but [here](https://gist.github.com/hemanth/163ca365868bb2086ee2) is the exported file.

**Comment by dvenkatsagar** [comment]
Hmmm, for some reason, when I wrote a code object(clicking on the code button and writing it) in the slide.com and export it, its giving me the correct format.

Im sorry but I think it must be a bug some where.
Its working for me, check this out [http://slides.com/dvenkatsagar/deck#/](http://slides.com/dvenkatsagar/deck#/)

**Comment by hemanth** [comment]
@dvenkatsagar #pain to convert all of them to those format.

**Comment by dvenkatsagar** [comment]
Well then there is nothing I can do.... Slides is working fine, Even reveal is showing it correctly.

**Comment by hemanth** [comment]
@dvenkatsagar If you noticed the gist I passed the code snippets in the slides are all blank and the quotes are broken.

**Comment by demoneaux** [comment]
This probably isn't the place to discuss as slid.es is a commercial product with their own processes and code which are in some cases different from Reveal. Perhaps asking on stackoverflow or their support channels would get you better answers. Closing this for now.
